backend/src/services/gameService.ts

```diff
@@ -124,16 +124,20 @@ export const filterGames = async (filter: Filter) => {
 	});
 };
 
-export const removeGame = async (gameID: string) => {
+export const removeGame = async (gameID: string, userGroup: string) => {
```

Comment:
Could we have a better variable name than `userGroup` as it sounds like a group but is actually a game owner id?